ush/python/pygfs/task/archive.py

```diff
@@ -75,7 +72,7 @@ def configure(self, arch_dict: Dict[str, Any]) -> (Dict[str, Any], List[Dict[str
         archive_parm = os.path.join(arch_dict.PARMgfs, "archive")
 
         # Collect the dataset to archive locally
-        arcdir_j2yaml = os.path.join(archive_parm, "arcdir.yaml.j2")
+        arcdir_j2yaml = os.path.join(archive_parm, f"{arch_dict.RUN}_arcdir.yaml.j2")
```

Comment:
I think you want to use `NET` instead of `RUN` since `RUN` can also be `gdas`, `enkfgdas`, or `enkfgfs`.  You will also have to add `NET` to the `keys` list in `exglobal_archive.py`.  `NET` is either `gfs` or `gefs` and later will include `sfs`.

```suggestion
        arcdir_j2yaml = os.path.join(archive_parm, f"{arch_dict.NET}_arcdir.yaml.j2")
```

Reply:
@DavidHuber-NOAA Thanks, will work on it.